prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -111,7 +111,16 @@ public async Task<EnrolleeViewModel> GetEnrolleeAsync(int enrolleeId)
                 .DecompileAsync()
                 .SingleOrDefaultAsync();
 
-            return _mapper.Map<EnrolleeViewModel>(dto);
+            var linkedGpid = await _context.EnrolleeLinkedEnrolments
+                .Where(ele => ele.EnrolleeId == enrolleeId)
+                .Select(ele => ele.UserProvidedGpid)
+                .SingleOrDefaultAsync();
+
+            dto.UserProvidedGpid = linkedGpid;
+
+            var vm = _mapper.Map<EnrolleeViewModel>(dto);
+            vm.UserProvidedGpid = linkedGpid;
+            return vm;
```

Comment:
you don't need to assign the value to both the the dto and the vm. When you call `_mapper.Map<EnrolleeViewModel>(dto);` it copies the value from the dto to the vm.

Reply:
you can just do `return _mapper.Map<EnrolleeViewModel>(dto);`